book/4 - MultiWorlds/1 - Multiworlds Overview.html

```diff
@@ -0,0 +1,21 @@
+<h4>Multiple worlds</h4>
+
+<p>
+	Cuberite supports multiple worlds. Each world has its own `world.ini` file. Additional Worlds can be added by editing <code>server.ini</code>. See the example below.
```

Comment:
Perhaps my wording was wrong. A good way to put it would be `This is
explained in the example below:`.

On 23/12/15 20:25, Safwat Halaby wrote:
> In book/4 - MultiWorlds/1 - Multiworlds Overview.html
> <https://github.com/cuberite/users-manual/pull/57#discussion_r48376799>:
> 
>> @@ -0,0 +1,21 @@
>> +<h4>Multiple worlds</h4>
>> +
>> +<p>
>> +	Cuberite supports multiple worlds. Each world has its own `world.ini` file. Additional Worlds can be added by editing <code>server.ini</code>. See the example below.
> 
> Any pointers? I can't think of a better way to explain that. Sometimes
> an example is the simplest.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cuberite/users-manual/pull/57/files#r48376799>.
> 

-- 

Alexander Harkness (bearbin)
Find my GPG key on the Ubuntu Keyserver - http://keyserver.ubuntu.com:11371/
Fingerprint: 93B3 D9D4 E65A CB47 BAF6 57AF 9537 B229 C1FD E7DF